src/_apis/image/get-image-url.ts

```
@@ -0,0 +1,36 @@
+          method: 'POST',
+          body: formData,
+        },
+      );
+      if (!response.data) {
+        throw new Error('Failed to get image: No data received');
+      }
+
+      return response.data;
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)